array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,14 @@
+/* exported equal */
+// iterate over the indexes of the first array
+// if index of first array is not equal to index of second array or if they aren't equal length,
+//  return false
+// otherwise, return true
+
+const equal = (first, second) => {
+  for (var i = 0; i < first.length; i++) {
+    if (first[i] !== second[i] || first.length !== second.length) {
+      return false;
+    }
```

Comment:
It's easier to understand the code if you compare the lengths of the arrays before the loop even starts.